showmehow/lessons.json

```diff
@@ -126,7 +126,7 @@
                 "task": "Gibberish again. Nothing bad will happen, I promise!",
                 "effects": {
                     "success": {
-                        "reply": "See? Harmless. Don't be afraid to try new things. You can either keep typing gibberish or you can enter 'showmehow' to head back to the main menu.",
+                        "reply": "See? Harmless. Don't be afraid to try new things. You can either keep typing gibberish or you can type 'showmehow' to the top-level of this tutorial.",
```

Comment:
I think you removed a bit too much.
`type 'showmehow' to head back to the top-level of this tutorial.`